File: serwer/src/main/java/pl/nakicompany/ewzwierzat/domain/Zwierze.java
package pl.nakicompany.ewzwierzat.domain;

import lombok.Data;

import javax.persistence.*;

@Entity
@Table(name = "zwierze")
@Data
public class Zwierze {
    @Id
    @Column(name="id")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(name="gatunek")
    private String gatunek;

    @Column(name="rodzaj")
    private String rasa;

    @Column(name="imie")
    private String imie;

    @Column(name="wiek")
    private Integer wiek;

    @Column(name="image_url")
    private String imageUrl;

    @Column(name="opis")
    private String opis;

    @Column(name="numer_czip")
    private String numerCZIP;

    @Column(name="cechy_szczegolne")
    private String cechySzczegolne;

    @Column(name="adoptowany")
    private Boolean adoptowany = false;
}

File: web/src/app/model/zwierze.ts
export class Zwierze {
  public gatunek: string;
  public rasa: string;
  public imie: string;
  public wiek: number;
  public nr: number;
  public img: string;
  public opis: string;

  constructor() {
    this.nr = 12;
    this.gatunek = 'kot';
    this.imie = 'Halinka';
    this.rasa = 'Nebelung';
    this.wiek = 4;
    this.img = '';
  }
}
